packages/integration/lib/src/widgets/user/user_created_post_listview.dart.dart

```diff
@@ -527,6 +530,218 @@ class _LMFeedUserCreatedPostListViewState
     );
   }
 
+  Widget _defPollWidget(LMFeedPoll pollWidget, LMPostViewData postViewData) {
```

Comment:
return type

Reply:
here return type will be `Widget` since it is origially called in `LMFeedPostMedia` where the return type is `LMFeedPollWidget` on feed_screen and other screens we are just ovverriding them with it's implementation